server/container/cli-integration/src/test/java/org/apache/james/cli/SieveCommandsIntegrationTest.java

```diff
@@ -121,4 +127,23 @@ public void removeSieveQuotaShouldWork() throws Exception {
         sieveProbe.getSieveQuota();
     }
 
+    @Test
+    public void addActiveSieveAndGetActiveScript() throws Exception {
+        File scriptFile = File.createTempFile("testscript", ".sieve");
+        scriptFile.deleteOnExit();
+
+        String script = "require \"fileinto\";\n" +
+                "\n" +
+                "fileinto \"INBOX.any\";";
+        FileUtils.writeStringToFile(scriptFile, script, StandardCharsets.UTF_8);
+
+        ServerCmd.doMain(new String[] {"-h", "127.0.0.1", "-p", "9999", CmdType.ADDSIEVESCRIPT.getCommand(),
+                USER, "myscript.sieve", scriptFile.getAbsolutePath()});
+
+        ServerCmd.executeAndOutputToStream(new String[] {"-h", "127.0.0.1", "-p", "9999",
+                CmdType.GETACTIVESIEVESCRIPT.getCommand(), USER}, outputCapture.getPrintStream());
+
+        assertTrue(outputCapture.getContent().equals(script));
```

Comment:
We try to use assertj `Assertions.assertThat` as much as possible because it's easier to read. Could you do that change ?